packages/design-system/src/components/Table/TableRow.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export interface TableRowProps {
+  /**
+   * The table row contents, usually `TableCell`.
+   */
+  children?: React.ReactNode;
```

Comment:
Essentially the same question as https://github.com/CMSgov/design-system/pull/1088/files#r674172226. A `tr`s only permitted children are supposed to be `<th>` or `<td>`, also `<script>` and `<template>`.

Reply:
Yes, permitted children are supposed to be <th> or <td>, also <script> and <template>. 
Do you think the current type definition for `children` is appropriate?